src/utils/proxy.ts

```diff
@@ -58,12 +61,48 @@ const decompressResponseBody = async function (body: Buffer, contentEncoding = '
     case 'br':
       return await brotliDecompress(body)
     case 'deflate':
-      return await deflate(body)
+      return await inflate(body)
```

Comment:
`inflate` is for decompressing.
`deflate` is for compressing.

Reply:
Why is the case `deflate`?

Reply:
@smnh I think the PR is ready to merge, just curious about this last point. If `case "deflate": return inflate` is intentional I think adding a code comment about why we handle the "deflate" case with "inflate" is a good idea here